src/activity_actor.cpp

```diff
@@ -1858,7 +1858,8 @@ void move_items_activity_actor::do_turn( player_activity &act, Character &who )
         }
 
         // Check that we can pick it up.
-        if( !target->made_of_from_type( phase_id::LIQUID ) ) {
+        if( !target->made_of_from_type( phase_id::LIQUID ) &&
+            !target->made_of_from_type( phase_id::GAS ) ) {
```

Comment:
I'd probably invert checks like this and just check for solid items, considering plasma would be equally unhandleable. Though I'm not sure if PNULL could ever show up in these cases and what it would even mean. It'd probably be a bug anyway and shouldn't be treated as if it was solid.

Reply:
I agree that plasma is an equally unsuitable material for handling, although I'm unsure if we actually have anything in the game in that form.

made_of_from_type seems to be doing strange things in regards to phase_id, especially for spilt frozen liquids. I would agree that it should probably just check for solid, but based on what I reviewed+tested I'll hold off on doing so until I get a chance to test it, probably sometime tomorrow.

Reply:
`made_of_from_type` is the default phase of the `itype`. Use `made_of` instead if you want the phase the item is currently in.

Plasma in fact only seems to be used by fields and not items. But who knows what the future brings.

Reply:
Ah yes, that is indeed the difference between those two. As soon as I had closed VS it all went straight out of my head. Opening it fresh today and reviewing your comment cleared it right up.